.github/workflows/validate-nginx-config.yml

```diff
@@ -0,0 +1,78 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      custom_template_values:
+        default: '{}'
+        description: 'Custom template values in JSON format to be passed to the environment variables. The key and value can be used in the Nginx configuration template files.'
+        required: false
+        type: string
+      port:
+        default: 8080
+        description: 'Nginx container port number.'
+        required: false
+        type: number
+      templatedir: # Ref: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/entrypoint/20-envsubst-on-templates.sh#L32
+        description: 'Nginx configuration template directory. Usually have files with .template extension.'
+        required: true
+        type: string
+      verbose:
+        default: true
+        description: 'Toggle verbose output. By setting this to true, the job will print out the nginx version, nginx config files, and generated config files.'
+        required: false
+        type: boolean
+      workdir:
+        default: 'web'
+        description: 'Default working directory to run the job.'
+        required: true
+        type: string  
+
+jobs:
+  validate-nginx-config:
+    name: validate-nginx-config
+    runs-on: ubuntu-latest
+    env:
+      PORT: ${{ inputs.port }}
```

Comment:
At this point, do you think that we should move the port to `custom_template_value`?